app/models/advanced_setting.rb

```diff
@@ -1,6 +1,9 @@
 class AdvancedSetting < ApplicationRecord
   belongs_to :resource, :polymorphic => true
 
+  supports(:update) { _("Can not edit read only model") if read_only? }
+  supports(:delete) { _("Can not delete read only model") if read_only? }
```

Comment:
- Not sure on using the word "model" here, because that is a developer term which probably doesn't make sense to an end user.  What is the wording currently like in the UI?
- `Can not` -> `Cannot`

Reply:
I was worried that this would add too many translations if they were all different. and didn't think the exception wording mattered.

I will fix